client/ayon_core/hosts/max/api/plugin.py

```diff
@@ -203,6 +250,31 @@ def create_instance_node(node):
         return node
 
 
+class MaxCacheCreatorBase(MaxCreatorBase):
```

Comment:
This class name feels too generic for what seems to be a TyFlow specific creator?